week-3/E-array-map/exercise.js

```diff
@@ -3,3 +3,14 @@
 
 var numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
 
+// var number100 =numbers.map(double);
+
+// function double(numbers) {
+//     return numbers * 100;
+// }
+// console.log(number100);
+
+var number100 = numbers.map(function (num) {
+    return num * 100;
```

Comment:
This is correct, but the exercise asks you to write multiple solutions using different syntax. Can you try that?

Reply:
@40thieves the lines commented out above from line number 6 to line number 11 is another method, please do take a look